docker/entrypoint.sh

```diff
@@ -79,6 +79,11 @@ then
     ETCD_CLUSTER="127.0.0.1:4001"
 fi
 
+mkdir -p ~postgres/.config/patroni
+cat > ~postgres/.config/patroni/patronictl.yaml <<__EOF__
+{dcs_api: 'etcd://${ETCD_CLUSTER}', namespace: /service/}
+__EOF__
+
```

Comment:
Did you mean `.json` instead of `.yaml` here?  Also, just below these lines we have `.yml` vs. `.yaml`.
And that doesn't look like correct JSON to me either: you need to double quote the keys and values, unless it is any of the `null`, `true`, `false` or numeric.